tutorials/other/ssh_keys.md

```diff
@@ -0,0 +1,91 @@
+Copyright 2015 [Nicky Semenza](http://nickysemenza.com)
```

Comment:
@leviathan747 wrote this in the Contribution Guide google doc:
![image](https://cloud.githubusercontent.com/assets/707582/6998378/48ae2590-dbaf-11e4-894f-2ae06ba0bb16.png)